### PR TITLE
OBPIH-3184 Minor improvements to budget code required for locations

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

**Review by awalkowiak — APPROVED** [review]

**Comment by awalkowiak** [comment]
@jmiranda I reviewed Dominika's part and pushed my changes to PO UI (because this branch is related to that ticket), if you are ok with everything then please merge.

File: grails-app/controllers/org/pih/warehouse/order/OrderController.groovy [hunk 1]
```diff
@@ -354,8 +356,10 @@ class OrderController {
                 if (orderAdjustment.orderItem && !params.orderItem.id) {
                     orderAdjustment.orderItem.removeFromOrderAdjustments(orderAdjustment)
                 }
-                orderAdjustment.properties = params
                 def budgetCode = BudgetCode.get(params.budgetCode?.id)
+                params.remove("budgetCode.id")
+                params.remove("budgetCode")
+                orderAdjustment.properties = params
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/order/OrderController.groovy:359]: Why do we need to do all this stuff with the params map? Line 362 above should be enough to handle budget code. Remove 359-361 and 363 and then let me know what happens.

**Review by jmiranda — CHANGES_REQUESTED** [review]

File: grails-app/controllers/org/pih/warehouse/order/OrderController.groovy (hunk 1 above)
Comment (jmiranda) [reply grails-app/controllers/org/pih/warehouse/order/OrderController.groovy:359]: I made the changes myself in both places.

Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/order/OrderController.groovy:359]: @jmiranda It was a fix to the issue Katarzyna mentioned in the Jira ticket. I agree it was not pretty. I fixed it in a different way and pushed it here https://github.com/openboxes/openboxes/pull/1927

**Pull request merged**